test/integration/custom_executable_test.rb

```diff
@@ -24,7 +26,7 @@ class CustomExecutableTest < Minitest::Test
       end
 
       test "'packwerk check' with violations fails and displays violations" do
-        Tempfile.create(["timeline_comment", ".rb"], timeline_path("app", "models")) do |file|
+        File.open(timeline_path("app", "models", "timeline_comment.rb"), "w") do |file|
```

Comment:
Should we maybe create a helper for this type of operation to simplify it? 

```ruby

# application helper module
  def open_app_file(*path)
      expanded_path = to_app_path(File.join(*path))
      File.open(expanded_path, "w") { |file| yield file }
  end

# test 

open_app_file(timeline_path("app", "models", "timeline_comment.rb")) do |file|
  # ...
end
```

Reply:
I like it.